recipe/meta.yaml

```
@@ -21,16 +22,16 @@ requirements:
     - poetry-core >=1,<2
   run:
     - python
-    - typing_extensions >=4.7.1  # [py<38]
+    - typing_extensions >=4.1,<5  # [py<310]
 
 test:
   imports:
     - graphql
   source_files:
     - tests
   requires:
-    - pytest >=6.2,<7
-    - pytest-benchmark >=3.4,<4
+    - pytest >=6.2
```

```suggestion
    - pytest >=6.2,<9
```
see: https://github.com/graphql-python/graphql-core/blob/b7935f75511dde280325631541f4a8a24b245709/tox.ini#L56C5-L56C19